src/components/Accordion/AccordionHeader.module.css

```diff
@@ -31,12 +52,19 @@
   border-right-style: var(
     --component-accordion_header_title-border_right_style
   );
+  text-align: var(--component-accordion_header_title-text-align);
   background-color: var(--component-accordion_header_title-color-background);
   font-size: var(--component-accordion_header_title-font_size);
   font-weight: var(--component-accordion_header_title-font_weight);
   margin-left: var(--component-accordion_header_title-spacing-margin_left);
 }
 
 .accordion-header__actions {
-  margin-top: 0.3rem;
+  margin: 0.3rem;
```

Comment:
I didnt notice this before, but this will also make the size of the header bigger when actions are included, vs no actions are included. This should probably also be removed? Or perhaps changed to `padding: 0 0.3rem` to not increase vertical size, but keep spacing on left/right.

Reply:
It will yeah.
Here, I just made sure the increase was at least symmetrical, because it looked strange as it was.
Technically, depending on what the user of the component injects into this, it might change the size anyway.
But I do agree. I'll change it to `padding: 0 0.3rem` since that looks nicest😊